src/pages/MetricDetail.svelte

```diff
@@ -264,7 +264,33 @@
         </tr>
       </table>
     {/each}
+  {:else}
+    <table>
+      <col />
+      <col />
+      <tr>
+        <td> Not Sampled </td>
+        <td>
+          This metric is not currently affected by any sampling configuration
+          and is operating based on its defaults in the metrics.yaml file
+        </td>
+      </tr>
+    </table>
   {/if}
+  <table>
+    <col />
+    <col />
+    <tr>
+      <td> Sampling Configuration Snippet </td>
+      <td>
+        Click the link to copy a configuration snippet for this metric to the
+        clipboard
```

Comment:
Could we provide some information about
a) What this configuration would do if we used it
b) What to do with it after we copy it

Reply:
I added to the text indicating what could be done with this snippet, along with linking to the Server Knobs documentation and the Experimenter documentation.